servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -484,155 +452,5 @@ public final ServerContext listenBlockingStreamingAndAwait(
      * @return A {@link Single} that completes when the server is successfully started or terminates with an error if
      * the server could not be started.
      */
-    public final Single<ServerContext> listenBlockingStreaming(final BlockingStreamingHttpService service) {
-        return listenForAdapter(toStreamingHttpService(service, strategyInfluencer(service)));
-    }
-
-    /**
-     * Starts this server and returns the {@link ServerContext} after the server has been successfully started.
-     * <p>
-     * If the underlying protocol (e.g. TCP) supports it this should result in a socket bind/listen on {@code address}.
-     *
-     * @param connectionAcceptor {@link ConnectionAcceptor} to use for the server.
-     * @param service {@link StreamingHttpService} to use for the server.
-     * @param strategy the {@link HttpExecutionStrategy} to use for the service.
-     * @param drainRequestPayloadBody if {@code true} the server implementation should automatically subscribe and
-     * ignore the {@link StreamingHttpRequest#payloadBody() payload body} of incoming requests.
-     * @return A {@link Single} that completes when the server is successfully started or terminates with an error if
-     * the server could not be started.
-     */
-    protected abstract Single<ServerContext> doListen(@Nullable ConnectionAcceptor connectionAcceptor,
```

Comment:
For any public methods that you remove from this API, consider submitting a PR for 0.41 branch to mark those `@Deprecated`. For cases when we do not provide an alternative API (like here), consider clarifying that users have to do it themselves. Example: #1791.

Reply:
Please check https://github.com/apple/servicetalk/pull/1830